src/hooks/useDisableScrolling.ts

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from 'react'
+import { isMobile } from 'utils/userAgent'
+
+export default function useDisableScrolling(open: boolean | undefined | null) {
```

Comment:
maybe rename this input to "disable" since `open` isn't related to anything internal to this function

Reply:
also this file seems like a great candidate for a unit test - see the `renderHook` function in test-utils and examples of its usage

Reply:
created tests!